wordpress-mobile/WordPress-Android
# All time widget configuration setup

Since Views widget and All-time widget have the same configuration, I'm reusing the configuration fragment and viewmodel. The only difference is the title (which comes from the separate configuration activities) and the action to update the widget. Both the color and the selected site are stored per `appWidgetId` which is unique. The All-time widget has a different placeholder icon.

The widget updated class is not yet implemented (it's copy/paste of the views updater). I'll implement that in the next PR.

To test:
* Add a widget 
* Notice there is new widget with a different placeholder icon
* Place it on the homescreen
* The configuration opens
* The title is "All-time"
* The color and site selector work as expected

Update release notes:
